docs/UserGuide.md

```diff
@@ -99,9 +99,10 @@ Edit at your own risk of losing data.
 
 
 **:information_source: Notes about the input format:**<br>
+* For `[p/Phone]`, a minimum of 3 digits must be inputted but most phone numbers would be at least 8 digits long.
 
-* For `[t/TAG]...`, only alphanumeric inputs are allowed. i.e. Only the characters A-Z, a-z, 0-9.<br>
-  e.g. `t/Based In Singapore` is not allowed, `t/BasedInSingapore` is allowed.
+* For `[t/TAG]...`, only alphanumeric inputs are allowed and cannot be empty. i.e. Only the characters A-Z, a-z, 0-9.<br>
+  e.g. `t/Based In Singapore` or `t/` is not allowed, `t/BasedInSingapore` is allowed.
```

Comment:
We might need to add a not that they cannot put t/High or t/applied

Reply:
Good shout! Made an additional note about this in the same subsection. Also reordered the points so that it reads better.